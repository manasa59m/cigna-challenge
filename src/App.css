.app-header {
  display: flex;
  align-items: center;
  padding: 16px;
  background-color: #e91e63;
  color: #fff;
}

.app-title {
  margin: 0;
}