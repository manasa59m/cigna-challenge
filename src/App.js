import './App.css';
import HomePage from './pages/HomePage';

function App() {
  return (
    <div className="app">
      <header className="app-header">
        <h1 className="app-title">CIGNA Challenge</h1>
      </header>
      <HomePage />
    </div>
  );
}

export default App;
