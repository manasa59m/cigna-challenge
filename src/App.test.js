import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import App from "./App";

describe("App Component", () => {
  it("should render without crash", () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("should contain title", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find(".app-title").text()).toBe("CIGNA Challenge");
  });
});
