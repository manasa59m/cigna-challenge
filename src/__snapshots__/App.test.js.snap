// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render without crash 1`] = `
<div
  className="app"
>
  <header
    className="app-header"
  >
    <h1
      className="app-title"
    >
      CIGNA Challenge
    </h1>
  </header>
  <HomePage />
</div>
`;
